vars/sectionNightlyTests.groovy

```diff
@@ -49,6 +49,52 @@ def call(PipelineCallbacksRunner pcr, AppPipelineConfig config, PipelineType pip
         }
       }
     }
+    if (config.parallelCrossBrowserTest) {
+      stageWithAgent("Cross browser tests", product) {
+        warnError('Failure in parallelCrossBrowserTest') {
+          Set<String> browsers = config.parallelCrossBrowsers.collect{ it.toLowerCase() }.toSet()
+          parallel(
+//            browsers.each {browser ->
```

Comment:
remove commented out code if you aren't using it

Reply:
I'm hoping to test 2 ways of running these in parallel - the current one where the browsers being called are fixed,and then the commented out way where the services will be able to override the default browsers with their own requirements (but I'm not sure i have this correct yet)

I'll definitely remove whichever one doesn't make the cut 👍